test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy

```diff
@@ -820,16 +820,25 @@ class LeVADocumentUseCaseSpec extends SpecHelper {
         then:
         def e = thrown(RuntimeException)
         e.message == "Error: Jira query returned 0 issues: '${jqlQuery}'."
+    }
+
+    def "notify LeVA document issue with query returning more than 1 issue"() {
```

Comment:
@hrcornejo Name should be changed

Reply:
Modify per: "notify LeVA document issue with query returning 3 issues"